src/components/header/ProfileHeaderButton.js

```diff
@@ -30,7 +30,7 @@ export default function ProfileHeaderButton() {
     >
       <Centered>
         <ContactAvatar
-          color={accountColor || colors.skeleton}
+          color={(isNaN(accountColor) && colors.skeleton) || accountColor}
```

Comment:
this should be a ternary instead of an &&

isNan(accountColor) ? colors.skeleton : accountColor